src/utils/D3D12Binding.cpp

```diff
@@ -188,32 +218,70 @@ namespace utils {
 
                 lastSerialRenderTargetWasUsed[renderTargetIndex] = backend::d3d12::GetSerial(backendDevice);
 
-                // Tell the backend to render to the current render target
-                backend::d3d12::SetNextTexture(backendDevice, renderTargetResources[renderTargetIndex]);
+                return NXT_SWAP_CHAIN_NO_ERROR;
+            }
+
+            // C stubs for C++ methods
+
+            static void Init(void* userData, void* wsiContext) {
```

Comment:
nit: we could have a templated helper that handles making these function and filling the nxtSwapChainImplementation so that we avoid doing it in every Binding file.